src/mapml/features/featureGroup.js

```diff
@@ -19,8 +19,11 @@ export var FeatureGroup = L.FeatureGroup.extend({
       if(layers.length === 1 && firstLayer.options.link) this.options.link = firstLayer.options.link;
       if(this.options.link){
         M.Feature.prototype.attachLinkHandler.call(this, this.options.group, this.options.link, this.options._leafletLayer);
+        this.options.group.setAttribute('role', 'link');
       } else {
         this.options.group.setAttribute("aria-expanded", "false");
+        // fix issue https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/423
+        this.options.group.setAttribute('role', 'application');
```

Comment:
I think still that `role="button"` is better suited in this case. I'm not going to object to using `application`, but just saying... for the record.

Reply:
Not sure if `role="button"` fixes https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/423 though. So maybe that's why. OTOH since we're currently using `role="application"` on the map element that fixes that issue.

Reply:
Updated